driver-embedded/src/main/com/mongodb/embedded/client/MongoDBCAPI.java

```diff
@@ -25,18 +26,28 @@
 
 
     // CHECKSTYLE.OFF: MethodName
+    /**
+     * The logging callback interface that handles logs from the mongodbcapi library.
+     */
+    interface LoggingCallback extends Callback {
```

Comment:
The `LoggingCallback` interface  represents the `libmongodbcapi_log_callback` used by the `libmongodbcapi_init_params`:

```
typedef void (*libmongodbcapi_log_callback)(
    void* user_data, const char* message, const char* component, const char* context, int severity);
```